### DISCUSSION TITLE
Arcade Control Board Drawing

### DISCUSSION THREAD

**Discussion by CooperTrout** [frc5567/2023-Robot#147] [Team Posts]
![unnamed](https://user-images.githubusercontent.com/35350257/52180358-6c3bd300-27b3-11e9-86f5-5375900e9b60.jpg)

**Comment by MikasaSucasa** [comment]
@CooperTrout Can you post a pdf of this updated CAD drawing? I want to confirm with @coverbeek that we have all functionality covered

**Comment by coverbeek** [comment]
https://github.com/frc5567/2019-Robot/blob/master/doc/ROBOGameControl.pdf
Cooper couldn't post it directly, so we checked it into a doc folder in the repo.
